### DISCUSSION TITLE
adding and changing monitoring interfaces in SO 2

### DISCUSSION THREAD

**Discussion by azxqw** [Security-Onion-Solutions/securityonion#2506] [Unsupported Versions] [answered]
Hi all,
I'm having issues with checking and changing the monitor interface post installation.
I have a suspicion i chose the wrong interface while installing, but I cant find where and how to check and change my monitoring interface without reinstalling SO
any help would be appreiciated

**Comment by TOoSmOotH** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/discussions/2450
